### PR TITLE
SUS-5672 | publish Slack notifications on production deployments

### PR DESCRIPTION
**Pull request by macbre**
https://wikia-inc.atlassian.net/browse/SUS-5672

### CODE REVIEW

File: docker/prod/Jenkinsfile [hunk 1]
```diff
@@ -5,6 +5,9 @@ def loggerImage = "artifactory.wikia-inc.com/sus/mediawiki-logger"
 
 def rolloutStatus = 1
 
+// Slack notifications
+env.MW_PUSH_CHANNEL="#mw-prod-deployment"
```
Comment (rogatty) [inline docker/prod/Jenkinsfile]: I think we need to set `env.MW_SANDBOX_K8S_WEBHOOK` too but this is a secret URL. Those variables should probably be passed from Jenkins.

Comment (macbre) [reply docker/prod/Jenkinsfile]: This one is defined in Jenkins (as a "global" env variable there).

Comment (rogatty) [reply docker/prod/Jenkinsfile]: So how about using this instead?
```
def slackChannel = "#mw-prod-deployment"
```
It's not an environment variable, after all.

Comment (macbre) [reply docker/prod/Jenkinsfile]: Good point - e3feb5a

**Review by rogatty — APPROVED** [review]

**Pull request merged**